### ISSUE TITLE
feature request: OpenStreetMap instead of Google Maps

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#71]
```
Wouldn't it be possible to make use of OSM when typing 'where' command, at 
least with a special setting?

Land marks for mapnik could be set by adding 'm' to words lat and lon, e.g. 
http://www.openstreetmap.org/?mlat=51.5073&mlon=-0.1278&zoom=14&layers=M
```

Original issue reported on code.google.com by `me.detected` on 2 Feb 2011 at 8:59

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 2 Feb 2011 at 9:02
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 3 Feb 2011 at 8:02
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 3 Feb 2011 at 8:28
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.9
```

Original comment by `Florent....@gmail.com` on 6 Feb 2011 at 8:27
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
